src/main/java/org/tron/p2p/connection/business/handshake/HandshakeService.java

```diff
@@ -20,10 +24,11 @@ public void startHandshake(Channel channel) {
   @Override
   public void processMessage(Channel channel, Message message) {
     HelloMessage msg = (HelloMessage) message;
-    log.debug("Receive message from {}, {}", channel.getInetSocketAddress(), message);
+    log.info("Receive message from {}, {}", channel.getInetSocketAddress(), message);
```

Comment:
It has been printed repeatedly and can be deleted.

Reply:
It is deleted.